### PR TITLE
Names refactor

### PR DESCRIPTION
**Pull request by mplebanski**
It's an important step to simplify current task API. I removed mappings that caused different identifiers point to the same data/objects. 

This change requires `golem-electron` to use `subtasks_count` instead of `subtasks` when creating/getting a task through golem RPC. @mdtanrikulu

### CODE REVIEW

**Comment by mplebanski** [comment]
I resolved the conflicts that appeared because it was hanging for over a week.

File: golem/client.py [hunk 1]
```diff
@@ -1,1900 +1,1900 @@
-# pylint: disable=too-many-lines
```
Comment (jiivan) [inline golem/client.py]: What happened here? Please revert this change.

**Review by jiivan — CHANGES_REQUESTED** [review]
Full body of `golem/client.py` changed...

File: golem/client.py (hunk 1 above)
Comment (etam) [reply golem/client.py]: Looks like change of end-of-line encoding.

@mplebanski I know we have some files with `\r\n` and some with `\n`, but we have to stick with what we have.

**Comment by mplebanski** [comment]
@jiivan I fixed the line endings problem. Weird.

File: golem/client.py (hunk 1 above)
Comment (mplebanski) [reply golem/client.py]: This happened because I used github web gui to merge the golem/client.py into feature branch

**Comment by etam** [comment]
@mplebanski when merging this PR (not now, obviously) please use the option to squash all commits.

**Comment by mplebanski** [comment]
Can we give it a go? It's easy for this refactor to become a conflict (many files affected).

**Review by Wiezzel — COMMENTED** [review]
This seems to affect RPC (`comp.task.create`). Do we have coordinated interface changes?

**Comment by mplebanski** [comment]
I only plan to do this regarding Electron client (@mdtanrikulu). Is there anything else I should take into consideration?

**Review by jiivan — APPROVED** [review]

**Comment by jiivan** [comment]
> I only plan to do this regarding Electron client (@mdtanrikulu). Is there anything else I should take into consideration?

Yes, have a look at *golemcli*

**Comment by mplebanski** [comment]
Closing due to too many conflicts

**Pull request closed without merging**